### PR TITLE
Ei 279 vrf status

### PR DESCRIPTION
**Pull request by MichaelYoung1981**

### CODE REVIEW

File: src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/SFA.DAS.EmployerIncentives.Api.AcceptanceTests.csproj [hunk 1]
```diff
@@ -69,6 +74,10 @@
   </ItemGroup>
 
   <ItemGroup>
+    <SpecFlowFeatureFiles Update="Features\GetVrfCaseDetailsForNewSubmissionsJob.feature">
+      <Generator>SpecFlowSingleFileGenerator</Generator>
```
Comment (ruslanss) [inline src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/SFA.DAS.EmployerIncentives.Api.AcceptanceTests.csproj:78]: don't get why these keep getting added 🤷‍♂️

File: src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/TestContext.cs [hunk 2]
```diff
@@ -26,5 +28,11 @@ public TestContext()
             TestData = new TestData();
             HashingService = new HashingService.HashingService("46789BCDFGHJKLMNPRSTVWXY", "SFA: digital apprenticeship service");
         }
+
+        public T GetApiService<T>()
+        {
+            var scope = ApiFactory.Services.CreateScope();
```
Comment (ruslanss) [inline src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/TestContext.cs:34]: lovely jubbly

File: src/SFA.DAS.EmployerIncentives.Commands/GetVrfCaseDetailsForNewSubmissions/GetVrfCaseDetailsForNewSubmissionsCommand.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿using SFA.DAS.EmployerIncentives.Abstractions.Commands;
+
+namespace SFA.DAS.EmployerIncentives.Commands.GetVrfCaseDetailsForNewSubmissions
+{
+    public class GetVrfCaseDetailsForNewSubmissionsCommand : ICommand
```
Comment (Stephek) [inline src/SFA.DAS.EmployerIncentives.Commands/GetVrfCaseDetailsForNewSubmissions/GetVrfCaseDetailsForNewSubmissionsCommand.cs:5]: This just seems wrong.  A Get command?

**Comment by ruslanss** [comment]
i broke this branch 😬

**Pull request closed without merging**